packages/vulcan-core/lib/modules/containers/withMulti.js

```diff
@@ -241,49 +236,47 @@ const queryReducer = (previousResults, action, collection, mergedTerms, listReso
   const mingoQuery = new Mingo.Query(selector);
 
   // function to remove a document from a results object, used by edit and remove cases below
-  const removeFromResults = (results, document) => {
-    const listWithoutDocument = results[listResolverName].filter(doc => doc._id !== document._id);
-    const newResults = update(results, {
-      [listResolverName]: { $set: listWithoutDocument }, // ex: postsList
-      [totalResolverName]: { $set: results[totalResolverName] - 1 } // ex: postsListTotal
+  const removeFromResults = (data, document) => {
+    const listWithoutDocument = data[resolverName].results.filter(doc => doc._id !== document._id);
```

Comment:
Is there a possibility of this being `null` on `results`?

Reply:
Yes, that's probably a possibility.